### PR TITLE
Add Product Manager Button and Update K2 Comment Buttons UI

### PR DESCRIPTION
**Pull request by grgia**
A branch from https://github.com/Expensify/k2-extension/pull/155
That adds the new button as well as updates the UI for comments buttons.

Fixes https://github.com/Expensify/Expensify/issues/277401

https://user-images.githubusercontent.com/38015950/235197459-297a9457-fef9-4ef0-be5d-7e3e169c7dab.mov

(out dated styles, but same comment behavior)

https://user-images.githubusercontent.com/38015950/234272979-628a68ec-37a5-426c-b59d-a1ff26daf0df.mov

### CODE REVIEW

**Comment by grgia** [comment]
@tgolen I branched from Lauren's branch [here](https://github.com/Expensify/k2-extension/pull/155), how would you suggest I handle merging these two PRs? Is it okay to close one or merge them?

File: src/js/module/K2comments/CommentsButtons.js [hunk 1]
```diff
@@ -17,50 +45,73 @@ class CommentsButtons extends React.Component {
...
+                    )}
                 </BtnGroup>
-
-                <BtnGroup>
+                {(this.state.isOpen && this.state.isButtonSelected && !this.state.shouldShowConfirmationMessage) && (
                     <button
                         type="button"
-                        className="btn btn-sm"
-                        onClick={() => this.addParticipationComment('I have read and reviewed this Project Manager Application!')}
+                        className="btn btn-sm send"
```
Comment (tgolen) [inline src/js/module/K2comments/CommentsButtons.js:101]: Let's add `btn-primary` to turn it green so it's not so easy to miss

**Review by tgolen — CHANGES_REQUESTED** [review]

**Comment by tgolen** [comment]
> Actually, I realize I'd have to close the first branch regardless

Yeah, closing the other one is fine 👍

**Comment by grgia** [comment]
cc @tgolen this is ready for another round! 

Also quick Q- should the selection clear if you close the dropdown?

Current behavior: 
Only clear selection after clicking the selection, closing the dropdown doesn't affect the selection

https://user-images.githubusercontent.com/38015950/234527606-803161cb-75e5-461e-96e2-151c0ff33841.mov

Alternative behavior: 
Clear selection after clicking the selection, and closing the dropdown clears the selection

https://user-images.githubusercontent.com/38015950/234527624-7bb383ae-d9be-4a4d-ba25-e08187fd33e7.mov

**Comment by tgolen** [comment]
> Also quick Q- should the selection clear if you close the dropdown?

I think it's a tricky UX for sure. I actually think that neither proposals quite fix it. I had an idea that might help:

1. Click to open the drop-down
2. Selecting an option keeps the drop-down open and indicates which option was selected (maybe a green check icon, but that would mean we need to remove the green check icon for design docs, which I think is OK). You could also maybe change the border or background color of the option.
3. Once an option is selected, then the send button appears
4. Closing the drop-down clears all selections and hides the send button

**Comment by grgia** [comment]
@tgolen here's a few quick options to illustrate:

We could change the title of the comments shortcuts button when one is selected:

image

otherwise, we could your suggestion using something like a checkbox and never hide the other options:

image

But I don't think the current branch UX is that confusing, we could also do nothing

**Comment by tgolen** [comment]
> We could change the title of the comments shortcuts button when one is selected:

Oh, actually I love this one. Let's do it!

**Comment by grgia** [comment]
@tgolen updated!

**Review by tgolen — APPROVED** [review]

**Comment by tgolen** [comment]
Looks great, Thanks!

**Pull request merged**